pkg/minikube/cluster/cluster.go

```diff
@@ -740,34 +616,14 @@ func GetKubernetesEndpointsWithNamespace(namespace string) (endpointGetter, erro
 
 // EnsureMinikubeRunningOrExit checks that minikube has a status available and that
 // that the status is `Running`, otherwise it will exit
-func EnsureMinikubeRunningOrExit(api libmachine.API) {
+func EnsureMinikubeRunningOrExit(api libmachine.API, exitStatus int) {
```

Comment:
I do not quite understand as this function should os.Exit how returning an error would work?  I will leave for now, definitely worth discussing though.